clang/lib/Driver/ToolChains/SYCL.cpp

```diff
@@ -15,6 +15,222 @@ using namespace clang::driver::tools;
 using namespace clang;
 using namespace llvm::opt;
 
+// Struct that relates an AOT target value with
```

Comment:
I think this should be merged with upstream logic. Please see https://github.com/llvm/llvm-project/pull/133194/files

Thanks